packages/server/test/UserProfile.test.ts

```diff
@@ -0,0 +1,39 @@
+import { HttpStatus } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+
+import { Role } from '../src/shared';
+import { initE2eFixture } from './E2eFixture';
+import { accessTokenAsCookie } from './shared';
+
+const PATH = '/api/users/profile';
+
+describe(`${PATH}`, () => {
+  const fixture = initE2eFixture();
+
+  afterEach(async () => {
+    await fixture.db.userModel.deleteMany();
+  });
+
+  it('GET /', async () => {
+    // Given
+    const user = {
+      addressId: [],
+      favouriteRestaurants: [],
+      favouriteDishes: [],
+      orders: [],
+      profileCompleted: false,
+    };
```

Comment:
```suggestion
    const user = {};
```

O wszystkie pola, które wymieniasz powinna zadbać sama schema (i wypełnić je dokładnie tak jak Ty tutaj), także nie musisz ręcznie jeszcze raz tego podawać.

Reply:
rozumiem, dzięki !